recipes/rgi/meta.yaml

```diff
@@ -61,6 +61,7 @@ requirements:
     - bamtools >=2.5.1
     - bedtools >=2.28.0
     - bwa >=0.7.17
+    - openssl ==1.0.*
```

Comment:
This won't really solve the problem, the solution is to rebuild blast 2.6.0.

Reply:
Actually, the version of blast this is using is wrong.

Reply:
There is a comment in the file saying it requires that exact version of blast

Reply:
On the readme file on github it mentions 2.9.0, so I went with that in my PR.